src/functions-and-arrays.js

```diff
@@ -47,6 +152,30 @@ const wordsCount = [
   'matter'
 ];
 
+function howManyTimes(arr, word) {
+  if(arr.length === 0) {
+    return 0;
+    }
+  let numOfWords = 0;
+  for (let i = 0; i < arr.length; i++) {
+    if (arr[i] === word) {
+        numOfWords++;
+        return numOfWords;
+        }
+    else if (arr[i] !== word) {
+      
+      return 0;
+      }
+      else if (numOfWords === 5){
+        return 5;
+      }
+  
+      
+      } 
+
+}
+
+
```

Comment:
I can't make this one work as expected. We can go through it if you like? I would have probably: 

- created a variable to check for true or false
- run a for loop to compare the words in the array to the second argument 
- stop the loop and return the check variable depending on the answer

```
function doesWordExist(wordsFind, word) {
  let check = true;
  for (let i = 0; i < wordsFind.length; i++) {
    if (wordsFind[i] === word) {
      check = true;
      break;
      could do this with the .include method
    }
    check = false;
  }
  return check;
}

```

Reply:
Hey Tim, thanks a lot for your feedback. Was really helpfull. 
Yeah didn't declare the i's, thx for the hint. I wondered why it didn't work.
A lot became clearer this morning during the recap of the lab with Jan. I understood the concept of the for of loop and used it in the iterations.
Now the only thing where i would need help to go through would be the last bonus #8.(just copy pasted the solution from Jan)
Should we perhaps discuss it tomorrow afternoon?